train.py

```diff
@@ -59,11 +59,11 @@ def get_base_argument_parser():
     # logging
     parser.add_argument("--experiment_name", type=str, default="dev")
     parser.add_argument("--logs_path", type=str, default="log")
+    parser.add_argument("--save_k_top_val_models", type=int, default=1)
```

Comment:
I am referring to this second argument

Reply:
If you enter review mode you can see that line 62 and 66 have an argument that seems to serve the same purpose

Reply:
Forgot this was there! Will remove